src/pagination/pagination.ts

```diff
@@ -110,13 +136,24 @@ export class NgbPagination implements OnChanges {
   @Output() pageChange = new EventEmitter();
 
   /**
+   *  If true current page will always be displayed in the middle.
```

Comment:
Maybe reword as `Whether to rotate pages with more pages than maxSize. Current page will be in middle.`